package.json

```diff
@@ -13,5 +16,8 @@
       "type": "MIT",
       "url": "http://www.opensource.org/licenses/mit-license.php"
     }
-  ]
+  ],
+  "engines": {
+    "node": ">=6.4.0"
+  }
```

Comment:
This is breaking for the current major version simply by adding the engines field & `loader-utils`.

Using loader-utils enforces NodeJS 4.3 or newer, not that it's a problem but with this approach this is a semver `MAJOR`